src/bfx_tools/bwa.ml

```diff
@@ -175,8 +192,9 @@ let align_to_sam
       String.concat ~sep:" " [
         "bwa aln";
         "-t"; Int.to_string processors;
-        "-O"; Int.to_string configuration.Configuration.Aln.gap_open_penalty;
-        "-E"; Int.to_string configuration.Configuration.Aln.gap_extension_penalty;
+        "-O"; Int.to_string configuration.Configuration.gap_open_penalty;
+        "-E"; Int.to_string configuration.Configuration.gap_extension_penalty;
+        "-M"; Int.to_string configuration.Configuration.mismatch_penalty;
```

Comment:
:heart: that we use one parameter name for the same option that is carried by 2 different flags for no reason !